### ISSUE TITLE
Find Annotation and Knowledge Graphs to integrate

### ISSUE THREAD

**Issue by josiahseaman** [graph-genome/component_segmentation#41]
**Assignee:** Ali Haider Bangash
The first step is to identify what data could be integrated through a knowledge graph and what is available.  What did the other Hackathon teams accomplish?  What is available?  Information goes in this issue.  We're looking for information that relates to genetic variants of the virus:
* Structural annotations channel. Protein structure => codon table => sequence position. We could mark up pangenome positions related to known protein variants
* Gene Annotations: Possibly only need the reference gene annotation GFF, but it would be nice to have these positions in the graph genome context.  @subwaystation has ensured we have coordinate transforms that go both ways pangenome <-> reference genome coordinates, using faldo in RDF.
* Clinical data: Possibly the most important. If we have any knowledge of patient outcomes, and what region they're from, we could connect a strain of the virus (which will contain variants) to a patient outcome: how long in hospital, how long on ventilator, etc.  We don't necessarily need a viral sequence from that specific individual, but at minimum a probable association with a variant.
    * Human DNA variation data could also be used as in [UK Biobank article](https://www.bbc.com/news/health-52243605).
    * Technically, annotated a complete human pangenome is beyond our current scope in that gigabase genomes will put strain on our pipeline. It may be possible, however to make local graphs of key regions like HLA or MHC inside the Human genome.
* Phylogenetics: We're going to have a phylogenetic tree eventually https://github.com/graph-genome/Schematize/issues/58.  It'd be nice to link this with the "country" and "town" concepts in the knowledge graph.  What geographic or transmission data could we bring in?

**Comment by hhaider15** [comment]
- Clinical data
 [South Korea's CoVid 2019 patients 5 Year patient history](https://hira-covid19.net/)   The government of the Republic of Korea decided to share the world’s first de-identified COVID-19 nationwide patient data with domestic and international researchers. The data sets are collected and processed promptly, thanks to the Korean National Health Insurance System, covering the entire population across the nation.

**Comment by hhaider15** [comment]
Structural annotations: Very well done by Machine learning working group- [Complete genomes of the strains: labelled with the respective source](https://github.com/covid19-bh-machine-learning/master/blob/master/data/sars_cov_2_fixed.fasta) & [its metadata](https://github.com/covid19-bh-machine-learning/master/blob/master/data/sars_cov_2_fixed_meta.csv)

**Comment by hhaider15** [comment]
Gene annotations: [whole genome nucleotide data pulled from RVDB release 14 as labels. Metadata for human & non-human pathogen phenotypes](https://github.com/covid19-bh-machine-learning/master/blob/master/data/RVDB14_complete_coronavirus_fixed.fasta)

**Comment by hhaider15** [comment]
Structure annotations: [Amino acid sequence data for common cold CoV and SARS-COV-2 for M, E & S proteins](https://github.com/covid19-bh-machine-learning/master/blob/master/data/epitope_data/common_coronaviruses_and_sars2_M_E_N_Spike.fasta) with [metadata](https://github.com/covid19-bh-machine-learning/master/blob/master/data/epitope_data/common_coronaviruses_and_sars2_M_E_N_Spike.csv)

**Comment by hhaider15** [comment]
Genes & structural annotations: [Proteomics data](https://www.ebi.ac.uk/pride/archive/projects/PXD017710) & [MassIVE/CCMS Maestro+MSstats reanalysis of MSV000085096 / PXD017710 Proteome and Translatome of SARS-CoV-2 infected cells](https://massive.ucsd.edu/ProteoSAFe/result.jsp?task=334df9b4f1af4501bca0a2aa63278a7d&view=display_metadata_results&file=f.RMSV000000308%2F2020-03-22_nuno_334df9b4%2Fmetadata%2FMSV000085096_SARS-CoV-2_proteome_translatome.csv#%7B%22table_sort_history%22%3A%22_dyn_%23Condition_asc%22%7D)

**Comment by subwaystation** [comment]
Hi @hhaider15 !
Thanks for all the links. We could work with e.g. `.csv` or `.fasta`.

But what we had in mind are [SparqlEndpoints](https://www.w3.org/wiki/SparqlEndpoints) which we could query using [SPARQL](https://en.wikipedia.org/wiki/SPARQL). 

I think a good start would be http://yummydata.org/. And maybe you will finde some endpoints which are not listed there ;)
Please come back to me, if you have more questions.

**Comment by subwaystation** [comment]
@josiahseaman  and Phylogenetics: As far as I got it from the #public_sequence_resource group, they will pack the metadata also into a SPARQL endpoint. Part of the metadata will be a mandatory field for `collection_location`. For the list of the required metadata please visit https://github.com/arvados/bh20-seq-resource/blob/master/example/minimal_example.yaml.

**Comment by innamoratika** [comment]
Ali- Just wanted to introduce myself post-convo with @josiahseaman : I'll be working on the phylo side of things and we should touch base at some point regarding using universal IDs for genomes. We should have enough in the phylo tree that we can track provenance and pass that on to you!

**Comment by hhaider15** [comment]
Agreed. Apologies I was busy earlier. Shall be working on this, now.

**Comment by hhaider15** [comment]
Good to see you @innamoratika
